packages/taiko-client/driver/txlist_fetcher/blob.go

```diff
@@ -79,12 +85,58 @@ func (d *BlobFetcher) Fetch(
 	return nil, pkg.ErrSidecarNotFound
 }
 
-// sliceTxList returns the sliced txList bytes from the given offset and length.
-func sliceTxList(id *big.Int, b []byte, offset, length uint32) ([]byte, error) {
-	if offset+length > uint32(len(b)) {
-		return nil, fmt.Errorf(
-			"invalid txlist offset and size in metadata (%d): offset=%d, size=%d, blobSize=%d", id, offset, length, len(b),
-		)
+// FetchPacaya implements the TxListFetcher interface.
+func (d *BlobFetcher) FetchPacaya(
+	ctx context.Context,
+	tx *types.Transaction,
+	meta metadata.TaikoBatchMetaDataPacaya,
+) ([]byte, error) {
+	if meta.GetNumBlobs() == 0 {
+		return nil, pkg.ErrBlobUsed
```

Comment:
shouldnt this be `pkg.ErrNoBlobsUsed` ?

Reply:
Changed: https://github.com/taikoxyz/taiko-mono/pull/18746/commits/e0912fdf83207815d646c59a0e3a01607674d9e4